csrankings-m.csv

```diff
@@ -797,6 +797,7 @@ Mark Wallace 0001,Monash University,http://monash.edu/research/explore/en/person
 Mark Wineberg,University of Guelph,https://www.uoguelph.ca/computing/people/mark-wineberg,1IwCKpcAAAAJ
 Mark Yatskar,University of Pennsylvania,http://markyatskar.com,6RxMYNEAAAAJ
 Mark Yim,University of Pennsylvania,https://www.seas.upenn.edu/directory/profile.php?ID=107,6zm5XD8AAAAJ
+Mark Zhandry,Princeton University,https://www.cs.princeton.edu/~mzhandry,RSO5QEIAAAAJ

```

Comment:
Home page says on leave. Faculty on non-sabbatical leaves to industry are not eligible for inclusion.

Reply:
Indeed. My mistake. Removed.